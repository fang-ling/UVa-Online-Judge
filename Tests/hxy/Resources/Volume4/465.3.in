2147483648 * 01
1 + 02147483648
2147483648 * 02147483647
00 + 2147483647
