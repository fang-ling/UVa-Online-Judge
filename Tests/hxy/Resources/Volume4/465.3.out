2147483648 * 01
first number too big
result too big
1 + 02147483648
second number too big
result too big
2147483648 * 02147483647
first number too big
result too big
00 + 2147483647
